tests/PHPStan/Command/ErrorFormatter/data/WindowsNewlines.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace BaselineIntegration;
+
+class WindowsNewlines {
```

Comment:
moved the test into this new file, which uses windows line endings.
also added a .gitattributes file, which should make sure that the line-endings for this file are enforced to CRLF